### DISCUSSION TITLE
How to show scrollbars in Combobox

### DISCUSSION THREAD

**Discussion by NilsEvers** [unovue/reka-ui#1019] [Q&A]
I want to restrict the height of the `ComboBoxViewport` and allow the content to scroll.
This works but the scrollbars are hidden because the `ComboboxViewport` sets the `scrollbar-width` to `none`.
This behaviour is inline with the `ScrollArea` component however here I cannot use the `ScrollAreaScrollBar` component.

What is the best way to solve this?

**Comment by dsuryadi** [comment]
You can simply manually add scrollbar-width in the ComboboxViewport:
`<ComboboxViewport style="scrollbar-width: thin">`
